pkg/api/azenvtypes.go

```diff
@@ -154,26 +154,26 @@ var (
 	// AKSWindowsServer2019OSImageConfig is the aks-engine image based on Windows Server 2019
 	AKSWindowsServer2019OSImageConfig = AzureOSImageConfig{
 		ImageOffer:     "aks-windows",
-		ImageSku:       "2019-datacenter-core-smalldisk-2211",
+		ImageSku:       "2019-datacenter-core-smalldisk-2212",
 		ImagePublisher: "microsoft-aks",
-		ImageVersion:   "17763.3532.221102",
+		ImageVersion:   "17763.3650.221213",
 	}
 
 	// AKSWindowsServer2019ContainerDOSImageConfig is the aks-engine image based on Windows Server 2019
 	// configured with containerd
 	AKSWindowsServer2019ContainerDOSImageConfig = AzureOSImageConfig{
 		ImageOffer:     "aks-windows",
-		ImageSku:       "2019-datacenter-core-ctrd-2211",
+		ImageSku:       "2019-datacenter-core-ctrd-2212",
 		ImagePublisher: "microsoft-aks",
-		ImageVersion:   "17763.3532.221102",
+		ImageVersion:   "17763.3650.221213",
 	}
 
 	// WindowsServer2019OSImageConfig is the 'vanilla' Windows Server 2019 image
 	WindowsServer2019OSImageConfig = AzureOSImageConfig{
 		ImageOffer:     "WindowsServer",
 		ImageSku:       "2019-Datacenter-Core-with-Containers-smalldisk",
 		ImagePublisher: "MicrosoftWindowsServer",
-		ImageVersion:   "17763.3287.220806",
+		ImageVersion:   "17763.3650.221105",
```

Comment:
Should this be 221213?

Reply:
That VHD was published by the Windows team and was used the base for the aks-engine VHDs so it'll have a different publication timestamp.

Here's where we feed the image details into packer

https://github.com/Azure/aks-engine/blob/d22c9790163bd7a317faefbddaa6392bcedac08c/vhd/packer/windows-2019-vars.json#L1-L7

Reply:
cool, just checking! :)